samples/nrf_desktop/src/events/config_event.h

```diff
@@ -42,6 +44,7 @@ struct config_event {
 
 	enum config_event_id id;
 	u8_t data[REPORT_SIZE_USER_CONFIG - 1];
+	bool stored;
```

Comment:
I understand this member carries the information that value is different to what is stored on flash already.
Can we rename it to `update_needed` or something similar?

Reply:
This member is used to ignore recurring events. `config.c` accepts config_events from USB and BLE. It also loads information from flash and send config_events to motion and possibly others.

I wanted to avoid creating a second type of event called e.g. `config_load_event` with all field duplicated with `config_event`, so I simply added this flag.

Since you asked about it, I will add a comment in code.

Reply:
this is what I had in mind - you don't store settings as it is already there (it is being loaded from flash or it is the same as already stored).
can you rename the member to `update_needed`? I think it would be self explanatory
(you can add a comment too)

Reply:
Just noticed this one. Well `update_needed` is not the best name because we can **update** (apply user issued value to module, e.g. write to motion sensor registers) or **store** (write to flash). What about `store_needed`?